lms/djangoapps/certificates/tests/test_views.py

```diff
@@ -37,6 +37,7 @@
     BadgeAssertionFactory,
 )
 from lms import urls
+from util import organizations_helpers as organizations_api
```

Comment:
We could change the name, although the current format aligns with the milestones_api implementation, so I figured it would be better to keep them the same.  Beyond naming, however, I'm wondering if we should think about evolving this into an OrganizationService (see InstructorService, PR #9069).  This might be a good piece of follow-on work after the MVP rolls out.